src/modules/farm/mocks/farm.abi.factory.mock.ts

```diff
@@ -0,0 +1,15 @@
+import { FarmAbiFactory } from '../farm.abi.factory';
+import { FarmAbiServiceMock } from './farm.abi.service.mock';
+
+export class FarmAbiFactoryMock {
```

Comment:
You don't need this mock. We have mocks for each farm ABI service. In unit tests you can use directly the FarmAbiFactory as provider and also provide the mocked versions for the versioned Abi services

Reply:
Removed and updated the unit test